tests/test_printing.py

```diff
@@ -186,32 +186,32 @@ def create_model():
 }
 
 compose_kernel_param_print_string = """\
-name                                       class      transform    trainable    shape    dtype      value\n\
------------------------------------------  ---------  -----------  -----------  -------  -------  -------\n\
-Product.kernels[0].kernels[0].variance     Parameter  Softplus     True         ()       float64        1\n\
-Product.kernels[0].kernels[0].lengthscale  Parameter  Softplus     False        ()       float64        2\n\
-Product.kernels[0].kernels[1].variance     Parameter  Softplus     True         ()       float64        1\n\
-Product.kernels[0].kernels[1].lengthscale  Parameter  Softplus     False        ()       float64        2\n\
-Product.kernels[1].kernels[0].variance     Parameter  Softplus     True         ()       float64        1\n\
-Product.kernels[1].kernels[0].lengthscale  Parameter  Softplus     False        ()       float64        2\n\
-Product.kernels[1].kernels[1].variance     Parameter  Softplus     True         ()       float64        1\n\
-Product.kernels[1].kernels[1].lengthscale  Parameter  Softplus     False        ()       float64        2"""
+name                                       class      transform         trainable    shape    dtype      value\n\
```

Comment:
I would recommend to set config lower value to None, this test printing is going to be pain in the neck in general. Let's just constrain it to limited settings.

Reply:
I'm happy to set these tests back to lower value = None. But I would like to add an explicit test in for the case where it is not None and keep the more informative print string:
```
transform
----------------
Softplus + Shift
```
instead of the uninformative
```
transform
---------
Chain
```

Reply:
...though maybe this is still overkill